src/libbson/src/bson/bson-cmp.h

```diff
@@ -53,26 +70,25 @@ BSON_BEGIN_DECLS
  * recommended.
  */
 
-
-#define BSON_CMP_SET(op, ss, uu, su, us)                                              \
-   static BSON_INLINE bool BSON_CONCAT3 (bson_cmp_, op, _ss) (int64_t t, int64_t u)   \
-   {                                                                                  \
-      return (ss);                                                                    \
-   }                                                                                  \
-                                                                                      \
-   static BSON_INLINE bool BSON_CONCAT3 (bson_cmp_, op, _uu) (uint64_t t, uint64_t u) \
-   {                                                                                  \
-      return (uu);                                                                    \
-   }                                                                                  \
-                                                                                      \
-   static BSON_INLINE bool BSON_CONCAT3 (bson_cmp_, op, _su) (int64_t t, uint64_t u)  \
-   {                                                                                  \
-      return (su);                                                                    \
-   }                                                                                  \
-                                                                                      \
-   static BSON_INLINE bool BSON_CONCAT3 (bson_cmp_, op, _us) (uint64_t t, int64_t u)  \
-   {                                                                                  \
-      return (us);                                                                    \
+#define BSON_CMP_SET(op, ss, uu, su, us)                                                                   \
+   static BSON_INLINE bool BSON_GNUC_DEPRECATED BSON_CONCAT3 (bson_cmp_, op, _ss) (int64_t t, int64_t u)   \
```

Comment:
```suggestion
   static BSON_INLINE bool BSON_CONCAT3 (bson_cmp_, op, _ss) (int64_t t, int64_t u) BSON_GNUC_DEPRECATED   \
```
For consistency? (Deprecation macro at end of function declaration.)